cypress/plugins/index.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+const { MongoClient } = require('mongodb');
+const { createHash } = require('crypto');
+
+const seedPosts = require('../fixtures/posts/index.js');
+const seedComments = require('../fixtures/comments/index.js');
+const seedUsers = require('../fixtures/users/index.js');
+
+function hashLoginToken(loginToken) {
+  const hash = createHash('sha256');
+  hash.update(loginToken);
+  return hash.digest('base64');
+};
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+// eslint-disable-next-line no-unused-vars
+module.exports = (on, config) => {
+  on('task', {
+    async dropAndSeedDatabase() {
```

Comment:
I'm pretty confident I've fixed the probabilistic login failure problem I was having, following a changeover in where users get seeded. I haven't had it show up since in 100 or so individual test runs.